ops/model.py

```diff
@@ -691,6 +691,32 @@ def set_ports(self, *ports: Union[int, 'Port']) -> None:
         for protocol, port in desired - existing:
             self._backend.open_port(protocol, port)
 
+    def reboot(self, now: bool = False) -> None:
+        """Reboot the host machine, after stopping all containers hosted on the machine.
+
+        Normally, the reboot will only take place after the current hook successfully
+        completes. Use the ``now`` argument when multiple reboots are required, to
+        reboot immediately without waiting for the hook to complete, and to restart the
+        hook after rebooting.
```

Comment:
How about making this slightly more specific/direct. Perhaps:

```suggestion
        completes. Use ``now=True`` to reboot immediately without waiting for the
        hook to complete; this is useful when multiple restarts are required (Juju
        will re-run the hook after rebooting).
```